### PR TITLE
Add keyboard shortcut to enable log messages and history activity

### PR DESCRIPTION
**Pull request by vishichoudhary**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [x] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-1190](https://tickets.metabrainz.org/browse/PICARD-1190)

# Solution
Add keyboard shortcut for log messages and history activity. 

# Action

### CODE REVIEW

File: picard/ui/mainwindow.py [hunk 1]
```diff
@@ -501,9 +501,13 @@ def create_actions(self):
 
         self.view_log_action = QtWidgets.QAction(_("View Error/Debug &Log"), self)
         self.view_log_action.triggered.connect(self.show_log)
+        # TR: Keyboard shortcut for "View Error/Debug Log"
+        self.view_log_action.setShortcut(QtGui.QKeySequence(_("Ctrl+M")))
```
Comment (samj1912) [inline picard/ui/mainwindow.py]: You would want this to be `Ctrl+L` to correspond to the type hint in the action widget above.

**Comment by samj1912** [comment]
@vishichoudhary please add meaningful commit messages. Read pt. 3 of git workflow from https://github.com/metabrainz/picard/blob/master/CONTRIBUTING.md

File: picard/ui/mainwindow.py (hunk 1 above)
Comment (samj1912) [inline picard/ui/mainwindow.py]: You would want this to be `Ctrl+L` to correspond to the type hint in the action widget above.

**Comment by vishichoudhary** [comment]
@samj1912  i will try to make commit messages more meaningful from next time.

**Comment by samj1912** [comment]
@vishalchoudhary, why next time, you can simply amend your commit and force push. I am assuming you will have to rebase your previous commit anyway to accommodate the shortcut key change.

File: picard/ui/mainwindow.py (hunk 1 above)
Comment (vishichoudhary) [reply picard/ui/mainwindow.py]: CTRL-L is for keyboard shortcut for lookup

Comment (samj1912) [reply picard/ui/mainwindow.py]: You might want to change the key hint then. Is ctrl e and d also taken?

Comment (vishichoudhary) [reply picard/ui/mainwindow.py]: Both are free . Which one should i use?

Comment (vishichoudhary) [reply picard/ui/mainwindow.py]: Sorry about ctrl-d it's used for opening folder

File: picard/ui/mainwindow.py [hunk 2]
```diff
@@ -501,9 +501,13 @@ def create_actions(self):
 
         self.view_log_action = QtWidgets.QAction(_("View Error/Debug &Log"), self)
```
Comment (samj1912) [inline picard/ui/mainwindow.py]: Move the ampersand to E

**Comment by vishichoudhary** [comment]
@samj1912 done.

**Pull request merged**